tests/e2e/utils/pages/setup-mc/step-1-set-up-accounts.js

```diff
@@ -244,11 +256,13 @@ export default class SetUpAccountsPage extends MockRequests {
 	 * @return {import('@playwright/test').Locator} Get Google account card.
 	 */
 	getGoogleAccountCard() {
-		return this.getAccountCards( {
-			has: this.page.locator( '.gla-account-card__title', {
-				hasText: 'Google',
-			} ),
-		} ).first();
+		return this.page
+			.locator( '.components-card-body', {
+				has: this.page.locator( '.gla-account-card__title', {
+					hasText: 'Google',
+				} ),
+			} )
+			.first();
```

Comment:
May add a CSS class for this as well like the `getMCAccountCard` method below.

Reply:
👍